test/adapters/standalone.test.ts

```diff
@@ -943,4 +944,28 @@ describe('standalone adapter', () => {
 
     close();
   });
+
+  test('with DELETE method mutation', async () => {
```

Comment:
whoops this test should have been included in this PR 👉 https://github.com/jlalmes/trpc-openapi/pull/124